openedx/features/course_search/static/course_search/js/views/search_item_view.js

```diff
@@ -26,6 +26,22 @@
                 this.template = options.template;
             },
 
+            unitIcon: function() {
+                var icon = null;
+                switch (this.model.attributes.content_type) {
+                case 'Video':
```

Comment:
Please see [this comment](https://openedx.atlassian.net/browse/TNL-7139?focusedCommentId=482019). We're using `fa-tasks` for verticals, so we can do this here too, if the `content_type == 'Sequence'`.

Reply:
Added ``fa-tasks`` icon when ``content_type == 'sequence'``.